src/base/ApplicationECSAlbCodeDeploy.ts

```diff
@@ -81,6 +84,36 @@ export class ApplicationECSAlbCodeDeploy extends Resource {
     );
   }
 
+  /**
```

Comment:
Thoughts on using a private method? Can't really test the method but it's being black box tested in the tests above which I think should be okay.

Reply:
I think the tests are sufficient here :)